.github/workflows/test-asan.yml

```
@@ -1,4 +1,4 @@
-name: test-asan
+name: Test Asan
```

Nit: According to https://github.com/google/sanitizers/wiki/AddressSanitizer, the abbreviation is _ASan_.

```suggestion
name: Test ASan
```

Of course, a lot of people are likely to think that's a typo, so if you'd rather not bother, I'm OK with that.